apps/serialization/gateway.proto

```diff
@@ -297,7 +297,8 @@ syntax = "proto3";
     uint32 level_number = 3;
     repeated Unit units = 4;
     repeated CurrencyReward currency_rewards = 5;
-    uint32 experience_reward = 6;
+    uint32 experience_reward = 6;    
+    repeated CurrencyCost cost = 7;
```

Comment:
The names of this variable in the schema/DB and Protobuf messages differ, this will probably break. Also, let's use a more descriptive name like "attempt_cost" or "play_cost" to explain better what this represents!